### ISSUE TITLE
Does not work on Windows

### ISSUE THREAD

**Issue by EFanZh** [Eugleo/magic-racket#15]
When I was opening a Racket file, the following error was reported:

```
Caught exn:
current-load-relative-directory: path is not a complete path
  path: d%3A/

Caught exn in request "textDocument/documentSymbol"
hash-ref: no value found for key
  key: 'file:///d%3A/test.rkt

[Error - 7:53:21 AM] Request textDocument/documentSymbol failed.
  Message: internal error in method "textDocument/documentSymbol"
  Code: -32603 
```

**Comment by Eugleo** [comment]
Thanks for reporting! Could you tell me which shell are you using in the integrated terminal? I think you can find out in the settings, under `terminal.integrated.shell.windows`.

**Comment by EFanZh** [comment]
I did not configure this option. And the default value seems to be `C:\WINDOWS\System32\WindowsPowerShell\v1.0\powershell.exe`.

**Comment by Eugleo** [comment]
Could I further ask you to
1. Enable the logging in Magic Racket's preferences (set it to "verbose")
2. Reload the VS Code window
3. Reproduce the error
4. Send me the logs that show up in the Output > Racket language client (copy & paste them into some `.txt`)

Thanks!

**Comment by EFanZh** [comment]
Log: [log.txt](https://github.com/Eugleo/magic-racket/files/4579095/log.txt).

**Comment by Eugleo** [comment]
Should be fixed in v0.5.6 (will be live any moment now). I still encounter some issues with the LSP, but only with specific files, which leads me to think it's an issue with the server itself. I'll investigate.
